lib/src/tasks/test/cli.dart

```diff
@@ -37,6 +37,9 @@ class TestCli extends TaskCli {
         abbr: 'j',
         defaultsTo: '$defaultConcurrency',
         help: 'The number of concurrent test suites run.')
+    ..addOption('pub-serve',
+        defaultsTo: '$defaultPubServe',
+        help: 'Runs the tests using a pub server.')
```

Comment:
We'd probably want another argument, which could be used to specify the port of an existing pub server.

Should just pass any additional arguments along, so that consumers can specify the`test` CLI's `--pub-serve=<port>` argument directly?